### DISCUSSION TITLE
Some questions regarding key binding

### DISCUSSION THREAD

**Discussion by Jimmy-Z** [zed-industries/zed#21354] [Feature Requests]
For the record, I've just used zed for two days, using the Windows nightly build from @deevus.

* Is there a simple way for a "vim only" key binding?
If I set `"base_keymap": "None"` , with `"vim_mode": true`, I got no key bindings at all, I hope there's a way to achieve this other than manually disable them one by one.

* default binding `ctrl-:` for `editor::ToggleInlayHints` doesn't work.
    * I bond it to `ctrl-i` instead and it worked
    * I followed the instruction and disabled that default binding, but it's not reflected on the UI.

* In VSCode there's a setting to show inlay hints only when ctrl-alt is hold, this is more convenient for me than a toggle, is there a way to achieve this in Zed?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
